packages/interfaces/src/protocols.ts

```diff
@@ -28,6 +28,10 @@ export type IBaseProtocolSDK = {
   readonly numPeersToUse: number;
 };
 
+type StoreProtocolOptions = {
+  peer: string;
```

Comment:
Let's make it `peers` in the following PR

Reply:
We only use one peer for Store, unlike LightPush and Filter. When use `getPeers()` within Store, we still only use the 0th index from the array of peers returned.
`peers: string[]` would be applicable to LightPush and Filter

Reply:
https://github.com/waku-org/js-waku/issues/2175#issuecomment-2553555843

@danisharora099 , let's make it a follow up feature to not overcomplicate this PR.